doc/release-notes.md

```diff
@@ -260,6 +260,9 @@ in the Low-level Changes section below.
   blockhash is provided, check the mempool. 3. If no blockhash is
   provided but txindex is enabled, also check txindex.
 
+- The `bumpfee` RPC has now the option to specify the change address
+  which is used to identify which output will pay the fee bump.
+
```

Comment:
This should be in its own file `release-notes-15341.md`

Reply:
Please see discussion in https://github.com/bitcoin/bitcoin/pull/15243#discussion_r250765937.